### PR TITLE
[e2e oci] add tests for the help option

### PR DESCRIPTION
**Pull request by gvallee**
**Description of the Pull Request (PR):**

Add e2e tests specific to the help option for all the commands where applicable.
This PR only aims at improving the e2e coverage by implementing positive cases.

Note that this PR adds test for `singularity oci <subcmd> --help` commands; while existing code cover `singularity help oci <subcmd>`.

**This fixes or addresses the following GitHub issues:**

- Addresses #4088 

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Comment by jscook2345** [comment]
These may be a duplicate of the work done in `testHelpOciContent` in `e2e/help/help.go`.

**Comment by gvallee** [comment]
Mmmm that is really weird, the coverage tool did not pick them up at all. I will investigate.

**Comment by gvallee** [comment]
> Mmmm that is really weird, the coverage tool did not pick them up at all. I will investigate.

I confirm that the coverage tool does not pick it up at the moment.

**Comment by gvallee** [comment]
> > Mmmm that is really weird, the coverage tool did not pick them up at all. I will investigate.
> 
> I confirm that the coverage tool does not pick it up at the moment.

I get it now: it is the difference between `singularity help oci attach` and `singularity oci attach --help`. So they kind of do the same thing but they are different commands. I personally believe we should have both.

File: e2e/oci/oci.go [hunk 1]
```diff
@@ -332,6 +332,76 @@ func (c *ctx) testOciBasic(t *testing.T) {
...
+			name:          "update",
+			expectedRegex: `^Update container cgroups resources \(root user only\)`,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			c.env.RunSingularity(
+				t,
+				e2e.WithCommand("oci"),
```
Comment (WestleyK) [inline e2e/oci/oci.go:395]: Do you need the `e2e.AsSubtest(tt.name)` here?

**Review by WestleyK — COMMENTED** [review]
Looks good to me, just one comment.

**Review by WestleyK — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**